src/main/kotlin/no/nav/familie/ef/iverksett/oppgave/OpprettOppgaveForBarnService.kt

```diff
@@ -0,0 +1,99 @@
+                }
+            }
+        }
+    }
+
+    private fun opprettFødselsnummer(fødselsnummer: String): Fødselsnummer? {
+        return try {
+            Fødselsnummer(fødselsnummer)
+        } catch (ex: IllegalArgumentException) {
+            secureLogger.warn("Kunne ikke opprette Fødselsnummer objekt av barn med fødselsnummer=$fødselsnummer")
+            null
+        }
+    }
+
+    private fun opprettOppgaveForBarn(iverksett: Iverksett, beskrivelse: String) {
+        if (innhentDokumentasjonOppgaveFinnes(iverksett)) {
+            logger.info("Oppgave av type innhent dokumentasjon finnes allerede")
+            return
+        }
+        val oppgaveId = oppgaveClient.opprettOppgave(OppgaveUtil.opprettOppgaveRequest(iverksett,
+                                                                                       enhetForInnhentDokumentasjon(iverksett),
+                                                                                       Oppgavetype.InnhentDokumentasjon,
+                                                                                       beskrivelse))?.let { it }
+                        ?: error("Kunne ikke opprette oppgave for barn med behandlingId=${iverksett.behandling.behandlingId}")
+        logger.info("Opprettet oppgave med oppgaveId=$oppgaveId")
+    }
+
+    private fun innhentDokumentasjonOppgaveFinnes(iverksett: Iverksett): Boolean {
+        val finnOppgaveRequest = FinnOppgaveRequest(tema = Tema.ENF, aktørId = iverksett.søker.personIdent)
+        val oppgaveTyper = oppgaveClient.hentOppgaver(finnOppgaveRequest)?.map { it.oppgavetype }
+        return oppgaveTyper?.contains(Oppgavetype.InnhentDokumentasjon.name) ?: false
+    }
+
+    private fun barnBlirEttÅr(innenAntallUker: Long, referanseDato: LocalDate, fødselsdato: LocalDate): Boolean {
+        return barnErUnder(12L, referanseDato, fødselsdato)
+               && LocalDate.now().plusWeeks(innenAntallUker) >= fødselsdato.plusYears(1)
+    }
+
+    private fun barnBlirSeksMnd(innenAntallUker: Long, referanseDato: LocalDate, fødselsdato : LocalDate): Boolean {
+        return barnErUnder(6L, referanseDato, fødselsdato)
+               && LocalDate.now().plusWeeks(innenAntallUker) >= fødselsdato.plusMonths(6L)
+    }
+
+    private fun barnErUnder(antallMnd: Long, referanseDato: LocalDate, fødselsdato: LocalDate): Boolean {
+        return referanseDato <= fødselsdato.plusMonths(antallMnd)
+    }
+
+    private fun referanseDato(innenAntallUker: Long, sisteKjøring: LocalDate): LocalDate {
+        val periodeGap = ChronoUnit.DAYS.between(sisteKjøring, LocalDate.now()) - innenAntallUker * 7
+        if (periodeGap > 0) {
+            return LocalDate.now().minusDays(periodeGap)
+        }
+        return LocalDate.now()
+    }
+
+    private fun enhetForInnhentDokumentasjon(iverksett: Iverksett): Enhet {
+
+        /** TODO : Legg til sjekk for egenansatte (4483 hvis egenansatt) */
+
+        return Enhet("4489", "")
+    }
+
```

Comment:
Eget endepunkt i familie-integrasjoner for enhet: 

https://github.com/navikt/familie-integrasjoner/blob/f6dbe6b7618f6f2dbcbbb868695615d73dcf932c/src/main/java/no/nav/familie/integrasjoner/arbeidsfordeling/ArbeidsfordelingController.kt

Tipper denne er best:
```
    @PostMapping("/enhet/{tema}/med-relasjoner")
    fun hentBehandlendeEnhetForPersonIdentMedRelasjoner(@PathVariable(name = "tema")
```